app/models/lookup_keys/lookup_key.rb

```diff
@@ -203,6 +204,16 @@ def cast_default_value
     true
   end
 
+  def reset_override_params
+    if override_changed? && !override?
```

Comment:
nitpick: this can be moved to the `before_validation` and not get here at all unless override has changed.